massa-bootstrap/src/messages.rs

```diff
@@ -762,7 +826,7 @@ impl BootstrapClientMessageDeserializer {
     ) -> Self {
         Self {
             id_deserializer: U32VarIntDeserializer::new(Included(0), Included(u32::MAX)),
-            length_error_deserializer: U32VarIntDeserializer::new(Included(0), Included(100000)),
+            length_error_deserializer: U32VarIntDeserializer::new(Included(0), Included(u32::MAX)),
```

Comment:
Because in the serialization, no max length is given, this is to ensure that the behavior is the same on both sides.
If you want, we can move this value to a new `MAX_ERROR_LENGTH` constant, and check it at serialization side also